packages/manager/tools/component-rollup-config/package.json

```diff
@@ -68,9 +69,8 @@
     "eslint-formatter-pretty": "^3.0.1",
     "eslint-plugin-import": "^2.19.1",
     "eslint-plugin-markdown": "^1.0.0",
-    "mocha": "^6.2.1",
     "npm-run-all": "^4.1.5",
     "rollup-plugin-hypothetical": "^2.1.0",
```

Comment:
Since following file has been removed `packages/manager/tools/component-rollup-config/test/test.js`, then we can get rid of this devDependency:
```suggestion
```